Java 10 compatibility?

I think there may be a missing dependency when trying to run the latest code in the master branch on Java 10.

"Caused by: java.lang.ClassNotFoundException: javax.xml.bind.JAXBException"

NB: I'm not a java guy, but a cursory google search shows that [JAXB was removed in Java 9](https://stackoverflow.com/a/48987120) and needs to be added as a dependency.

I didn't see any mention of java version compatibility in the README...

--- a/pom.xml
+++ b/pom.xml
@@ -194,6 +194,18 @@
       <version>0.1</version>
     </dependency>
 
+    <dependency>
+        <groupId>jakarta.xml.bind</groupId>
+        <artifactId>jakarta.xml.bind-api</artifactId>
+        <version>2.3.3</version>
+    </dependency>
+
+    <dependency>
+        <groupId>org.glassfish.jaxb</groupId>
+        <artifactId>jaxb-runtime</artifactId>
+        <version>2.3.3</version>
+    </dependency>
+
   </dependencies>
 
   <build>

